apps/dashboard/Gemfile

```diff
@@ -46,6 +46,7 @@ end
 gem 'nokogiri', '~> 1.15', '< 1.16'
 gem 'net-imap', '~> 0.3', '< 0.4'
 gem 'public_suffix', '~> 5.0', '< 6.0'
+gem 'turbo-rails', '2.0.7'
```

Comment:
We have to lock `turbo-rails` because higher versions don't have Ruby 2.7 support.